tests/testsuite/directory.rs

```diff
@@ -189,7 +189,7 @@ error: failed to compile `bar v0.1.0`, intermediate artifacts can be found at `[
 Caused by:
   no matching package named `baz` found
 location searched: registry `https://github.com/rust-lang/crates.io-index`
-did you mean: bar, foo
+did you mean: bar, foo?
```

Comment:
I wonder if the `?` would look better here before the colon? I'm a little worried about it accidentally looking like it's part of the crate name

Reply:
I was thinking the same. 
- `did you mean: bar, foo ?`
-  `did you mean? bar, foo` 
- `Perhaps you mean: bar, foo`

Reply:
sounds good to me! although it may want to be worded "perhaps you meant: foo, bar"

Reply:
Yeah, `perhaps you meant: foo, bar` looks good.

Alternatively we can quote them: `did you mean: "bar", "foo"?`.

And, to be honest, I think we could also throw an "or" in there, so with 3 cases it would look like:

* `perhaps you meant: bar, foo, or baz`, or
* `did you mean: "bar", "foo", or "baz"?`

([Oxford comma](https://en.oxforddictionaries.com/explore/what-is-the-oxford-comma/) usage optional! 😉)

But I'd be happy to accept either of the first two.

Reply:
I like the idea of "or", although I'd prefer to avoid using quotes as it makes it easier to copy/paste without them sometimes

Reply:
Go with `perhaps you meant: bar, foo, baz`.
If you're feeling ambitious: `perhaps you meant: bar, foo, or baz` 😄

Reply:
Oh, I included quotes! Okey... Removing them in the sec..